pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -319,8 +319,14 @@ func handleEvent(s *Server) {
 
 	// Only updating intermediate CA is supported now
 	if !bytes.Equal(currentCABundle, newCABundle) {
-		log.Info("Updating new ROOT-CA not supported")
-		return
+		// Check if newCABundle or currentCABundle is combination of other
+		if bytes.Contains(currentCABundle, newCABundle) ||
+			bytes.Contains(newCABundle, currentCABundle) {
```

Comment:
What does it check?

Reply:
case 1: newCABundle is B+C
case 2: newCABundle is only C

Reply:
Is Istio able to successfully `Verify` a root cert containing multiple certs? Would be good to add test cases for `security/pkg/pki/util/keycertbundle.go`

Reply:
`UpdateVerifiedKeyCertBundleFromFile` will verify.